<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.Hcl.DirectCommerce.LoginPage.Verify_the_Login_InCorrect_User_Name_and_password" tests="1" time="32.624" ignored="0" failures="1" timestamp="23 Nov 2020 09:35:46 GMT" skipped="0" hostname="IN2-1025604LT" errors="0">
  <testcase name="VerifytheLoginInCorrectUserNameAndpassword" time="32.624" classname="com.Hcl.DirectCommerce.LoginPage.Verify_the_Login_InCorrect_User_Name_and_password">
    <failure type="java.lang.AssertionError" message="An error occured while logging into portal, check if the credentials are correct.">
      <![CDATA[java.lang.AssertionError: An error occured while logging into portal, check if the credentials are correct.
	at org.testng.Assert.fail(Assert.java:94)
	at com.Hcl.DirectCommerce.uiActions.LoginPage.logintoApplication(LoginPage.java:133)
	at com.Hcl.DirectCommerce.LoginPage.Verify_the_Login_InCorrect_User_Name_and_password.VerifytheLoginInCorrectUserNameAndpassword(Verify_the_Login_InCorrect_User_Name_and_password.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- VerifytheLoginInCorrectUserNameAndpassword -->
</testsuite> <!-- com.Hcl.DirectCommerce.LoginPage.Verify_the_Login_InCorrect_User_Name_and_password -->
